.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python: "2.7"
+before_install:
+  - "export DISPLAY=:99"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
Are you planning to add integrations tests as well?  If not, ther's no point in starting xvfb before running the tests.